client/my-sites/marketing/connections/account-dialog.jsx

```diff
@@ -89,10 +90,13 @@ class AccountDialog extends Component {
 	}
 
 	areAccountsConflicting( account, otherAccount ) {
-		return (
-			account.keyringConnectionId === otherAccount.keyringConnectionId &&
-			account.ID !== otherAccount.ID
-		);
+		// Accounts are conflicting, if they have the same ID and keyringConnectionId, which
+		// means they are the same account. Otherwise it's a different site for the same account.
+		const isIdConflicting = isEnabled( 'jetpack-social/multiple-connections' )
+			? account.ID === otherAccount.ID
+			: account.ID !== otherAccount.ID;
```

Comment:
I'm not sure this would ever happen. If these IDs are the same then would that mean we've sent back a duplicate connection?

I think now we want to make sure that (if we can) we check the external ID for the connection. We don't want someone connecting the same Facebook page twice, but maybe we should handle that on the server and make sure that the site doesn't have two connections to the same external entity.

Reply:
Yeah it would mean that it's duplicated. For facebook, you cannot pick the same site twice, but if there is a site that let's you do that, I think we should make sure to show it as a conflict, as it was already connected. Do you think we should just not check for the ID in case of multiple connections?

Reply:
Updated to just return `false` for now as we agreed on the Slack huddle.